asmrun/Makefile

```diff
@@ -191,13 +197,13 @@ depend:
 	$(error Dependencies cannot be regenerated using the MSVC ports)
 else
 depend: $(COBJS:.$(O)=.c) $(sources)
-	$(CC) -MM $(CFLAGS) $(CPPFLAGS) $^ | sed -e 's/\.o/.$$(O)/' > .depend
-	$(CC) -MM $(PFLAGS) $(CPPFLAGS) $^ | sed -e 's/\.o/.p.$$(O)/' \
-	  >> .depend
-	$(CC) -MM $(DFLAGS) $(CPPFLAGS) $^ | sed -e 's/\.o/.d.$$(O)/' \
-	  >> .depend
-	$(CC) -MM $(IFLAGS) $(CPPFLAGS) $^ | sed -e 's/\.o/.i.$$(O)/' \
-	  >> .depend
+	$(CC) -MM $(OC_CFLAGS) $(OC_CPPFLAGS) $^ | sed -e 's/\.o/.$$(O)/' > .depend
+	$(CC) -MM $(OC_CPPFLAGS) $(OC_PROF_CPPFLAGS) $^ | \
+	  sed -e 's/\.o/.p.$$(O)/' >> .depend
+	$(CC) -MM $(OC_CPPFLAGS) $(OC_DEBUG_CPPFLAGS) $^ | \
+	  sed -e 's/\.o/.d.$$(O)/' >> .depend
+	$(CC) -MM $(OC_CPPFLAGS) $(OC_INSTR_CPPFLAGS) $^ | \
```

Comment:
In all three of these, I think it may be relevant to have had `PFLAGS`, `DFLAGS` and `IFLAGS` first - doesn't GCC go with the first option encountered? In other words, switching it around I think causes CPPFLAGS always to win?

(I'm not convinced this matters in practice, I'm just wondering if the change is necessary)